shaders/outlinepattern.fragment.glsl

```diff
@@ -21,5 +21,13 @@ void main() {
     vec2 pos2 = mix(u_pattern_tl_b, u_pattern_br_b, imagecoord_b);
     vec4 color2 = texture2D(u_image, pos2);
 
-    gl_FragColor = mix(color1, color2, u_mix) * u_opacity;
+    // find distance to outline for alpha interpolation
+    float dist_a = length(v_pos_a - gl_FragCoord.xy);
+    float alpha_a = smoothstep(1.0, 0.0, dist_a);
+
+    float dist_b = length(v_pos_b - gl_FragCoord.xy);
+    float alpha_b = smoothstep(1.0, 0.0, dist_b);
+    
+
+    gl_FragColor = mix(color1, color2, u_mix) * mix(mix(alpha_a, alpha_b, u_mix), u_opacity, 0.5);
```

Comment:
`alpha` and `u_opacity` should be combined with multiplication instead of `mix`. If the pattern opacity is already 0.5 and the alpha based on distance from the line is 0.5 the combined opacity should be 0.25